examples/main.cpp

```diff
@@ -0,0 +1,29 @@
+#include "ThingooConnector.h"
+
+#include <ESP8266HTTPClient.h>
+#include <ArduinoJson.h>
+#include <WiFiManager.h>
+#include <WiFiClientSecureBearSSL.h>
+
+const char* fingerprint = "68 E7 10 AD 43 DE EE 7E 24 67 71 BC 43 77 16 23 78 64 AF X5";
```

Comment:
Remove our instance details, this is a public library. Use some fake data

Reply:
It s a fake data :D

Reply:
device_id, host and secret_key surely aren't ;)